pxr/imaging/hgi/resourceBindings.h

```diff
@@ -130,6 +131,7 @@ struct HgiTextureBindDesc
     HgiBindResourceType resourceType;
     uint32_t bindingIndex;
     HgiShaderStage stageUsage;
+    bool readOnly;
```

Comment:
In shadersection we use `_writable` while here we use readOnly.  For consistency shouldn't they be the same?